core/settings.py

```diff
@@ -12,22 +12,25 @@
 
 from pathlib import Path
 import os
+import environ
+from django.conf.global_settings import INTERNAL_IPS
+
+env = environ.Env()
+environ.Env.read_env()
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
```

Comment:
It might be better to load environment file using explicit path.

```suggestion
env = environ.Env()

# Build paths inside the project like this: BASE_DIR / 'subdir'.
BASE_DIR = Path(__file__).resolve().parent.parent

# Load .env from the root of the project
environ.Env.read_env(BASE_DIR / ".env")
```